packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -114,22 +119,127 @@ class _MyHomePageState extends State<MyHomePage>
+              width: 16,
+            ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.folder_copy_outlined),
+              iconSize: 21,
+            ),
+            const SizedBox(
+              width: 21,
+            ),
+            const Text('Freelance'),
+          ],
+        ),
+        const SizedBox(
+          height: 16,
+        ),
+        Row(
+          children: [
+            const SizedBox(
+              width: 16,
+            ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.folder_copy_outlined),
+              iconSize: 21,
+            ),
+            const SizedBox(
+              width: 21,
+            ),
+            const Text('Mortage'),
+          ],
+        ),
+        const SizedBox(
+          height: 16,
+        ),
+        Row(
+          children: [
+            const SizedBox(
+              width: 16,
+            ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.folder_copy_outlined),
+              iconSize: 21,
+            ),
+            const SizedBox(
+              width: 21,
+            ),
+            const Flexible(
+                child: Text(
+              'Taxes',
+              overflow: TextOverflow.ellipsis,
+            )),
+          ],
+        ),
+        const SizedBox(
+          height: 16,
+        ),
+        Row(
+          children: [
+            const SizedBox(
+              width: 16,
+            ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.folder_copy_outlined),
+              iconSize: 21,
+            ),
+            const SizedBox(
+              width: 21,
+            ),
+            const Flexible(
+                child: Text(
+              'Receipts',
+              overflow: TextOverflow.ellipsis,
+            )),
+          ],
+        ),
+      ],
+    );
 
     // These are the destinations used within the AdaptiveScaffold navigation
     // builders.
-    const List<NavigationDestination> destinations = <NavigationDestination>[
+    const List<NavigationDestination> destinations = [
       NavigationDestination(
-          label: 'Inbox', icon: Icon(Icons.inbox, color: iconColor)),
+          label: 'Inbox', icon: Icon(Icons.inbox, color: Colors.black)),
       NavigationDestination(
           label: 'Articles',
-          icon: Icon(Icons.article_outlined, color: iconColor)),
+          icon: Icon(Icons.article_outlined, color: Colors.black)),
       NavigationDestination(
           label: 'Chat',
-          icon: Icon(Icons.chat_bubble_outline, color: iconColor)),
+          icon: Icon(Icons.chat_bubble_outline, color: Colors.black)),
       NavigationDestination(
           label: 'Video',
-          icon: Icon(Icons.video_call_outlined, color: iconColor)),
+          icon: Icon(Icons.video_call_outlined, color: Colors.black)),
```

Comment:
nit: These should use iconColor as their color. It is just closer to the example in the M3 spec.